lib/MusicBrainz/Server/Edit/Release/ChangeQuality.pm

```diff
@@ -88,5 +89,11 @@ method accept
     );
 }
 
+sub allow_auto_edit
+{
+    my $self = shift;
+    return $self->can_amend && $self->data->{new}{quality} != $QUALITY_HIGH;
```

Comment:
Reversing these will save a query against the edit table if the quality is high.